api/agentpb/collector.proto

```diff
@@ -313,6 +313,25 @@ message MetricsBucket {
     float m_cpu_sys_time_cnt = 29;
     // Total time system spent in query.
     float m_cpu_sys_time_sum = 30;
+    //
+    // pg_stat_monitor 0.9 metrics
+    //
+    // Total number of planned calls.
+    float m_plans_calls_sum = 31;
+    float m_plans_calls_cnt = 32;
+    // Total number of WAL (Write-ahead logging) records.
+    float m_wal_records_sum = 33;
+    float m_wal_records_cnt = 34;
+    // Total number of FPI (full page images) in WAL (Write-ahead logging) records.
+    float m_wal_fpi_sum = 35;
+    float m_wal_fpi_cnt = 36;
+    // Metrics skipped due to different bucket_time in pg_stat_monitor (1min in PMM, 5min in pg_stat_monitor):
```

Comment:
on https://www.percona.com/doc/percona-monitoring-and-management/2.x/setting-up/client/postgresql.html  we recommend the user set  

ALTER SYSTEM SET pg_stat_monitor.pgsm_bucket_time=60;

so this should not be a problem

Reply:
@rnovikovP I will create subtask to add also plan time.